app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryStateActions.js

```diff
@@ -40,11 +42,20 @@ class CollapsedStoryStateActions extends React.Component {
   }
 
   needConfirmation(action) {
-    return action === storyTransitions.ACCEPT || action === storyTransitions.REJECT;
+    return action === storyTransitions.ACCEPT ||
+      action === storyTransitions.REJECT ||
+      action === storyTransitions.RELEASE;
+  }
+
+  getStoryState(story) {
```

Comment:
What do you think about moving this logic to this file https://github.com/Codeminer42/cm42-central/pull/535/files#diff-b59c70dd5bb6cd4a0b5b67e0490f03fdR315 ?